packages/did-ethr-resolver/test/did-operator.test.ts

```diff
@@ -197,6 +197,26 @@ const testSuite = (): void => {
     )).not.undefined;
   });
 
+  it('It should not be possible to add service without service endpoint', async () => {
+    const attribute = DIDAttribute.ServicePoint;
+    const serviceId = 'UserClaimURL2';
+    const updateData: IUpdateData = {
+      type: attribute,
+      value: {
+        id: `${did}#service-${serviceId}`,
+        type: 'ClaimStore',
+      },
+    };
+    try {
```

Comment:
you can replace this with return expect(operator.update(...)).rejectedWith('Service Endpoint is required')